### ISSUE TITLE
Create kubernetes setup with 1 master 3 nodes

### ISSUE THREAD

**Issue by Madhu-1** [kshlm/gd2-k8s#2]
currently, I see once 3 vms are created in gd2-k8s cluster
`kube1`
`kube2`
`kube3`
is it meant to be 3 nodes and 1 of the node is considered a master?
if yes, can we separate out the master by creating a cluster of 1 master and 3 nodes setup?

**Comment by kshlm** [comment]
Why though? What do you get by having a separate master?

**Comment by Madhu-1** [comment]
it good to have a separate master as we should not disturb the workload on master VM. if we start bringing the containers(app pods) on the master node, even after some time the master components will go down due to the resource constraints.

**Comment by humblec** [comment]
I agree with @Madhu-1 , master is normally unschedulable. So its better to have one master and 3 worker nodes.

**Comment by kshlm** [comment]
Kubespray defaults to having 2 master nodes though. So the load gets shared.
But anyways, how big of a concern is this, just for a local development deployment? We are not saying that this is the preferred setup for production deployments.

**Comment by humblec** [comment]
@kshlm we are planning to make use of this for CSI driver e2e testing with all the driver functionalities. I expect a moderized load in such a setup.
